templates/algob-react-boilerplate/.eslintrc.js

```diff
@@ -0,0 +1,51 @@
+module.exports = {
+  env: {
+    browser: true,
+    node: true,
+    es6: true,
+    es2020: true,
+    mocha: true
+  },
+  plugins: ["simple-import-sort", "sonarjs"],
+  extends: [
+    "plugin:sonarjs/recommended",
+    'airbnb',
+    "eslint:recommended", // recommended rules
```

Comment:
```suggestion
    "eslint:recommended", 
    "plugin:react/recommended"
```

Reply:
We don't need the full `airbnb` - it's very opinionated. Let's user react instead: https://github.com/yannickcr/eslint-plugin-react

Reply:
OK. Thanks for suggestion.